ibm_catalog.json

```diff
@@ -562,8 +562,189 @@
+            },
+            {
+              "key": "wait_till"
+            },
+            {
+              "key": "wait_till_timeout"
+            },
+            {
+              "key": "log_analysis_enabled"
+            },
+            {
+              "key": "log_analysis_agent_tags"
+            },
+            {
+              "key": "log_analysis_add_cluster_name"
+            },
+            {
+              "key": "log_analysis_ingestion_key"
+            },
+            {
+              "key": "log_analysis_secret_name"
+            },
+            {
+              "key": "log_analysis_instance_region"
+            },
+            {
+              "key": "log_analysis_endpoint_type",
+              "options":[
+                {
+                  "displayname" : "private",
+                  "value" : "private"
+                },
+                {
+                  "displayname" : "public",
+                  "value" : "public"
+                }
+              ]
+            },
+            {
+              "key": "log_analysis_agent_custom_line_inclusion"
+            },
+            {
+              "key": "log_analysis_agent_custom_line_exclusion"
+            },
+            {
+              "key": "log_analysis_agent_name"
+            },
+            {
+              "key": "log_analysis_agent_namespace"
+            },
+            {
+              "key": "log_analysis_agent_tolerations"
+            },
+            {
+              "key": "cloud_monitoring_enabled"
+            },
+            {
+              "key": "cloud_monitoring_access_key"
+            },
+            {
+              "key": "cloud_monitoring_secret_name"
+            },
+            {
+              "key": "cloud_monitoring_instance_region"
+            },
+            {
+              "key": "cloud_monitoring_endpoint_type",
+              "options":[
+                {
+                  "displayname" : "private",
+                  "value" : "private"
+                },
+                {
+                  "displayname" : "public",
+                  "value" : "public"
+                }
+              ]
+            },
+            {
+              "key": "cloud_monitoring_metrics_filter"
+            },
+            {
+              "key": "cloud_monitoring_agent_tags"
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_name"
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace"
+            },
+            {
+              "key": "cloud_monitoring_agent_tolerations"
+            },
+            {
+              "key": "logs_agent_enabled"
+            },
+            {
+              "key": "logs_agent_name"
+            },
+            {
+              "key": "logs_agent_namespace"
+            },
+            {
+              "key": "logs_agent_trusted_profile"
```

Comment:
I think also make `logs_agent_trusted_profile` and `logs_agent_iam_api_key` required for same reason as https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/pull/167/files#r1787492298 = one of them is has to be passed